### PR TITLE
K8s 1522

### PR DESCRIPTION
**Pull request by bashofmann**

### CODE REVIEW

File: pages/02.Documentation/10.metakube-webterminal/default.en.md [hunk 1]
```diff
@@ -8,10 +8,16 @@ taxonomy:
 ---
 
 Every MetaKube cluster comes with a web-terminal directly in the MetaKube web interface that gives command line access to the cluster directly from the browser.
-To access the dashboard just click on the `Web Terminal` link on the cluster detail page. This bash terminal runs on Alpine Linux in the background and comes installed with all the necessary tools to interact with your cluster:
+To access the dashboard just click on the `Web Terminal` link on the cluster detail page:
+
+![MetaKube Dashboard](metakube-dashboard.png)
```
Comment (bashofmann) [inline pages/02.Documentation/10.metakube-webterminal/default.en.md:13]: This should be an image of the webterminal

File: pages/02.Documentation/10.metakube-webterminal/default.en.md [hunk 2]
```diff
@@ -8,10 +8,16 @@ taxonomy:
 ---
 
 Every MetaKube cluster comes with a web-terminal directly in the MetaKube web interface that gives command line access to the cluster directly from the browser.
-To access the dashboard just click on the `Web Terminal` link on the cluster detail page. This bash terminal runs on Alpine Linux in the background and comes installed with all the necessary tools to interact with your cluster:
+To access the dashboard just click on the `Web Terminal` link on the cluster detail page:
```
Comment (bashofmann) [inline pages/02.Documentation/10.metakube-webterminal/default.en.md]: "dashboard" is probably a copy&paste error here

File: pages/03.Tutorials/11.manage-worker-nodes-via-cli/default.en.md [hunk 3]
```diff
@@ -19,80 +19,89 @@ To get a list of all nodes execute:
...
+          operatingSystem: ubuntu
+          operatingSystemSpec:
+            distUpgradeOnBoot: false
+          sshPublicKeys: []
+      versions:
+        kubelet: 1.13.3
 ```
 
-For more details on Machines see [Cluster Management API](../../02.Documentation/12.cluster-api/default.en.md).
+The most common fields to update are:
```
Comment (bashofmann) [inline pages/03.Tutorials/11.manage-worker-nodes-via-cli/default.en.md]: There should be some note here, that the machine-controller is doing a rolling update of the deployment and how it does it (cordon, drain, recreate)

File: pages/03.Tutorials/09.manage-node-deployments/default.en.md [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+
+To add or delete a worker node you can easily edit the node deployment in your cluster. Navigate to the cluster overview and click on the node deployment you want to change. You will then see the node deployment overview. In this overview click on the `Edit Node Deployment` button:
+
+![Node deployment overview with highlighted edit button](image_edit_np_button_hightlight.png)
+
+In the popup you can now in- or decrease the number of worker nodes, which are managed by this node deployment:
+
+![Node deployment overview with opened edit modal](image_edit_np_modal.png)
+
+Save your changes and wait for MetaKube to add or delete your extra worker node(s):
```
Comment (bashofmann) [inline pages/03.Tutorials/09.manage-node-deployments/default.en.md:26]: There should be some note here, that the machine-controller is doing a rolling update of the deployment and how it does it (cordon, drain, recreate)

File: pages/03.Tutorials/09.manage-node-deployments/default.en.md [hunk 5]
```diff
@@ -0,0 +1,28 @@
...
+
+TODO
+
+## Edit the node deployment
+
+To add or delete a worker node you can easily edit the node deployment in your cluster. Navigate to the cluster overview and click on the node deployment you want to change. You will then see the node deployment overview. In this overview click on the `Edit Node Deployment` button:
+
+![Node deployment overview with highlighted edit button](image_edit_np_button_hightlight.png)
+
+In the popup you can now in- or decrease the number of worker nodes, which are managed by this node deployment:
```
Comment (bashofmann) [inline pages/03.Tutorials/09.manage-node-deployments/default.en.md]: "as well as the Kubelet Version, OS Image, ..)

File: pages/03.Tutorials/09.manage-node-deployments/default.en.md [hunk 6]
```diff
@@ -0,0 +1,28 @@
...
+
+![Node deployment overview with opened edit modal](image_edit_np_modal.png)
```
Comment (bashofmann) [inline pages/03.Tutorials/09.manage-node-deployments/default.en.md:24]: You can also add "tags" or "labels", with the interface. We can describe what this is (or link to the k8s docs for it)

File: pages/03.Tutorials/09.manage-node-deployments/default.en.md (hunk 4 above)
Comment (cbeneke) [reply pages/03.Tutorials/09.manage-node-deployments/default.en.md:26]: It's actually create->cordon&drain->delete ;)

File: pages/03.Tutorials/09.manage-node-deployments/default.en.md (hunk 5 above)
Comment (cbeneke) [reply pages/03.Tutorials/09.manage-node-deployments/default.en.md]: fixed

File: pages/02.Documentation/10.metakube-webterminal/default.en.md (hunk 1 above)
Comment (cbeneke) [reply pages/02.Documentation/10.metakube-webterminal/default.en.md:13]: Added it further below

File: pages/03.Tutorials/09.manage-node-deployments/default.en.md (hunk 4 above)
Comment (cbeneke) [reply pages/03.Tutorials/09.manage-node-deployments/default.en.md:26]: added a link to cluster api

**Comment by bashofmann** [comment]
lgtm

**Pull request merged**